pdal/pdal_types.hpp

```diff
@@ -190,6 +190,7 @@ namespace ClassLabel
     const uint8_t IgnoredGround = 20;
     const uint8_t Snow = 21;
     const uint8_t TemporalExclusion = 22;
+    const uint8_t Synthetic = 32;
```

Comment:
Add Withheld and Key-Point too?

Reply:
Oh, right. I'd thought about this before and then promptly forgot. I think I got disillusioned at the prospect of having to add separate arguments to the filters to support each. Perhaps we could change it from a boolean to a comma-separated string of bits to ignore, e.g., `ignore_bits="synthetic,withheld"`.

Reply:
@hobu Look at latest commit and let me know what you think.